torch/nn/modules/pixelshuffle.py

```diff
@@ -15,12 +15,15 @@ class PixelShuffle(Module):
     `Real-Time Single Image and Video Super-Resolution Using an Efficient Sub-Pixel Convolutional Neural Network`_
     by Shi et. al (2016) for more details.
 
+    Note that this function can take inputs with any number of batch dimensions:
```

Comment:
TODO: not for this PR but we should update the PixelShuffle and pixel_shuffle documentation. They start off by referring to a value named "r" for example, without ever defining what "r" is!